src/android/IndoorAtlasWayfinding/IAWayfinding.gradle

```diff
@@ -0,0 +1,15 @@
+repositories {
+    mavenCentral()
+    maven {
+        url "https://indooratlas-ltd.bintray.com/mvn-internal"
```

Comment:
Could we put these to the same `indooratlas.gradle` and not rename any of the existing files?